ros2topic/ros2topic/verb/pub.py

```diff
@@ -119,6 +123,7 @@ def main(args):
             times,
             args.wait_matching_subscriptions
             if args.wait_matching_subscriptions is not None else int(times != 0),
+            args.max_wait_time,
```

Comment:
how about checking the option dependencies before calling in here?

```python
if args.wait_matching_subscriptions is None and args.max_wait_time is not None:
    return '--max-wait-time option is only effective with --wait-matching-subscriptions'
```

maybe too much, cz we can actually ignore this `--max-wait-time` option.

Reply:
c6dda6e